src/index.js

```diff
@@ -261,6 +263,17 @@ class FlashlsHandler {
       updateSelectedIndex(this.qualityLevels_,
                           this.tech_.el_.vjs_getProperty('level') + '');
     }
+
+    onTrackChanged(this.tech_);
+    this.tech_.audioTracks().addEventListener('change', this.onAudioTrackChanged);
```

Comment:
it should be safe to use `this.tech_.audioTracks().on(...` instead of `addEventListener`